Learning_test_automation/Projekt#1/TeachersGradebook/SchoolActivity/StudentRepository.cs

```diff
@@ -15,32 +15,25 @@ public void AddStudent(IStudent student, StudentFile studentFile)
             studentFile.StudentFileCreation(student.ToString());
         }
 
-        public Student GetStudent(int id)
+        public IStudent GetStudent(int id)
         {
-            foreach(Student student in students) 
-            {
-                if (student.Id.Equals(id)) return student;
-            }
+            return students.FirstOrDefault(student => student.Id.Equals(id));
+
             throw new NullReferenceException("We do not have a student with this id.");
```

Comment:
Nieosiągalna linijka w kodzie

Reply:
no tak cokolwiek po return to ciężko żeby się wykonało xD